package no.kartverket.altinnpdp.client.exception

class AltinnException(
    message: String,
    statusCode: Int? = null,
    responseBody: String? = null,
    cause: Throwable? = null,
) : AltinnPdpException(message, statusCode, responseBody, cause)
